common-lib/src/main/scala/com/gu/mediaservice/lib/argo/ArgoHelpers.scala

```diff
@@ -24,6 +24,27 @@ trait ArgoHelpers extends Results {
     serializeAndWrap(response, Ok)
   }
 
+  def respondCollections[T](
+    uri: Option[URI] = None,
+    data: Map[String, Seq[T]],
+    links: List[Link] = Nil
+  ) (implicit writes: Writes[T]): Result = {
+    val collections = data.map({ case (key, collection) => {
+      key -> CollectionReponse(
```

Comment:
Embedded entities [must](https://github.com/argo-rest/spec#uri) have a `uri` parameter. Otherwise they are not embedded resources, if they don't have a `uri`. What's the main reason for wrapping them with data/length in this case?